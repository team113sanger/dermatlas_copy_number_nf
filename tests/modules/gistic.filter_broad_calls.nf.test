nextflow_process {

    name "Test Process FILTER_BROAD_GISTIC2_CALLS"
    script "modules/gistic2.nf"
    process "FILTER_BROAD_GISTIC2_CALLS"

    test("Should run without failures") {

        when {
        params {
            load("$baseDir/tests/testdata/test_params.json")
        }
            process {
                """
                input[0] = tuple([analysis_type: "test", plot_dir: "test"], file("$baseDir/tests/testdata/ascat_segments.tsv"))
                input[1] = tuple([analysis_type: "test", plot_dir: "test"], file("$baseDir/tests/testdata/broad_significance_results.txt"))
                input[2] = tuple([analysis_type: "test", plot_dir: "test"], file("$baseDir/tests/testdata/broad_values_by_arm.txt"))
                input[3] = file("$baseDir/tests/testdata/ucsc_chrom_arms.txt")
                input[4] = 0.1
                input[5] = "test"
                """
            }
        }

        then {
            assert process.success
        }

    }

}
