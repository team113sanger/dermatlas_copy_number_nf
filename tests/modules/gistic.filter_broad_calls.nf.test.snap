{
    "Should run without failures": {
        "content": [
            {
                "0": [
                    
                ],
                "cs": [
                    
                ]
            }
        ],
        "meta": {
            "nf-test": "0.8.4",
            "nextflow": "23.10.1"
        },
        "timestamp": "2024-06-19T10:56:20.435447"
    }
}